azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1671,6 +1671,11 @@ private void ajaxHandleUpload(final HttpServletRequest req, final HttpServletRes
     final User user = session.getUser();
     final String projectName = (String) multipart.get("project");
     final Project project = this.projectManager.getProject(projectName);
+    if (project.isActive() == false) {
```

Comment:
In CRT deployment, `uploadProject` will be called inside `azkaban.restli.ProjectManagerResource#deploy`. Do we also need to add the same fix there?

Reply:
Good found! Let me fix it.